packages/db/package.json

```diff
@@ -9,17 +9,21 @@
     "clean": "rm -rf .turbo node_modules",
     "lint": "biome check . --write",
     "lint-check": "biome check .",
-    "type-check": "tsc --noEmit"
+    "type-check": "tsc --noEmit",
+    "test": "vitest run"
   },
   "dependencies": {
     "@dotkomonline/logger": "workspace:*",
+    "@testcontainers/postgresql": "^10.13.2",
```

Comment:
These should be devdependencies

Also maybe consider dropping chalk, testfailure text diff is already red right?